types/puppeteer/index.d.ts

```diff
@@ -2025,27 +1968,21 @@ export interface LaunchOptions extends Timeoutable {
    * @default false
    */
   dumpio?: boolean;
-  /** Path to a User Data Directory. */
-  userDataDir?: string;
   /**
    * Specify environment variables that will be visible to Chromium.
    * @default `process.env`.
    */
   env?: {
     [key: string]: string | boolean | number;
   };
-  /**
-   * Whether to auto-open DevTools panel for each tab. If this option is true, the headless option will be set false.
-   */
-  devtools?: boolean;
   /**
    * Connects to the browser over a pipe instead of a WebSocket.
    * @default false
    */
   pipe?: boolean;
 }
 
-export interface DefaultArgsOptions {
+export interface ChromeArgOptions {
```

Comment:
Why rename?

Reply:
It was to align with the name used in the puppeteer source code: https://github.com/GoogleChrome/puppeteer/blob/v1.11.0/lib/Puppeteer.js#L32
https://github.com/GoogleChrome/puppeteer/blob/v1.11.0/lib/Launcher.js#L377

But I can revert it if you think it does not make sense.